tests/main/document-portal-activation/task.yaml

```
@@ -41,18 +41,29 @@ restore: |
     rm -rf "${XDG_RUNTIME_DIR:?}/*" "${XDG_RUNTIME_DIR:?}/.[!.]*"
 
 execute: |
+    #shellcheck source=tests/lib/systems.sh
+    . "$TESTSLIB"/systems.sh
+
+    check_stderr() {
+        test -n "$1"
+        if is_cgroupv2; then
+            [[ "$(cat "$1")" == "WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement" ]]
```

Why `[[`? Would this work?

```suggestion
            [ "$(cat "$1")" = "WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement" ]
```